kalsmic/iReporterApi
# 163847996 user view profile

**This pull request implements the profile page view for the user functionality:-**
Logic implemented
- Admin user can view statistics for records of all users on the application
- Non-admin users only view statistics for their own records they have created
- Also logged in user is able to view their personal user details on the profile page.

**user story** https://www.pivotaltracker.com/story/show/163847996

**How to test**
- log in  as an admin user and view  the profile page 
- log in as a normal user and view the profile page
- You will also notice that the statistics for admin users are for all the records on the application whereas those for non-admin users are for their own activity on the application.
- Also, the profile page contains details of the user who is currently logged in

**Links**
- UI :  https://kalsmic.github.io/iReporterApi/UI/index.html

**Credentials**
- Username: `arthur2` Password: `Password123`
- Usernmae: `admin` Password: `iReporter123`

@NicholusMuwonge @Rhytah @JamesMudidi @sanya-kenneth @frankopkusianwar @AtamaZack @ManuelDominic @KabohaJeanMark @nadralia @reifred @llwasampijja @richien @Genza999 @3Nakajugo @e-ian @Manorlds-Eaglespark @PatrickMugayaJoel @bisonlou @v1b3m @mwinel @sebbss, @kbjude
